src/routes/blog/labels.svelte

```diff
@@ -0,0 +1,26 @@
+<script lang="ts">
+  export let data: string[] = [];
+</script>
+
+{#if data.length > 0}
+  <div class="labels">
+    {#each data as label}
+      <small class="label">#{label}</small>
```

Comment:
Convert these label pills into links to the selected GitHub tag on top of the user's selected labels.

Reply:
Resolved by commit <https://github.com/EthanThatOneKid/acmcsuf.com/commit/1aad2bd45f6a3d87d102e4e16a6a98e85b7e498a>.